Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -4762,7 +4764,7 @@ AXTreeData AXObjectCache::treeData()
     if (isIsolatedTreeEnabled()) {
         stream << "\nAXIsolatedTree:\n";
         if (auto tree = getOrCreateIsolatedTree()) {
-            constexpr OptionSet<AXStreamOptions> options = { AXStreamOptions::ObjectID, AXStreamOptions::ParentID };
+            constexpr OptionSet<AXStreamOptions> options = { AXStreamOptions::ObjectID, AXStreamOptions::ParentID, AXStreamOptions::TextRuns };
```

Comment:
Do we want to commit this change?

Reply:
Ah no this was inadvertently included, but, maybe we should add this by default when `AX_THREAD_TEXT_APIS` is true? That can be for another day though...